config/README.md

```diff
@@ -2,85 +2,26 @@
 
 ## Intro
 
-HDF5 can be configured using both the GNU Autotools and CMake. We try to keep
-them in sync, but you can expect minor differences to crop up. Please create
-a GitHub issue for any differences noted. Note that with the Autotools, we
-do NOT check generated files into GitHub until release time, so you will
-need to generate `configure`, `Makefile.in`(s), etc. via `autogen.sh` in the
-project root if you want to build with that system.
+HDF5 can be configured using CMake.
 
 Configuration information for the HDF5 library and tools is (unfortunately)
 spread across the repository. Basic library configuration will generally
-be found in `configure.ac` (Autotools) and the root's `CMakeLists.txt` (CMake).
-Each subdirectory of the project also has its own `Makefile.am` or CMake build
+be found in the root's `CMakeLists.txt`.
+Each subdirectory of the project also has its own CMake build
 and test files.
 
 This directory contains a few important things:
 
-* Autotools OS- and compiler-specific configuration
 * CMake support files (in `cmake`)
-* Warning files shared between the two systems (in `*-warnings` directories)
+* Warning files (in `*-warnings` directories)
 * CMake toolchain files (in `toolchain`)
 * CMake sanitizer files (in `sanitizer`)
 
-CMake will be documented elsewhere. This document focuses on the Autotools files
-and the shared warning files.
+CMake is documented in the following files:
```

Comment:
Should it say "CMake is documented in the following files in the release_docs directory"?